IntelFsp2Pkg/Include/Ppi/FspiArchConfigPpi.h

```diff
@@ -0,0 +1,59 @@
+/** @file
+  Header file for FSP-I Arch Config PPI for Dispatch mode
+
+  Copyright (c) 2024, Intel Corporation. All rights reserved.<BR>
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef _FSPI_ARCH_CONFIG_PPI_H_
+#define _FSPI_ARCH_CONFIG_PPI_H_
+
+#define FSPI_ARCH_CONFIG_PPI_REVISION  0x1
+
+///
+/// Global ID for the FSPI_ARCH_CONFIG_PPI.
+///
+#define FSPI_ARCH_CONFIG_GUID \
+  { \
+    0x15735ef9, 0x84ac, 0x4e34, { 0x98, 0x86, 0x56, 0xee, 0xcf, 0x73, 0xec, 0x0f } \
+  }
+
+///
+/// This PPI provides FSP-M Arch Config PPI.
+///
+typedef struct {
+  ///
+  /// Revision of the structure is 1 for this version of the specification.
+  ///
+  UINT8                   Revision;
+  UINT8                   Reserved[3];
+  ///
+  /// Length of the structure in bytes. The current value for this field is 64.
+  ///
+  UINT32                  Length;
+  ///
+  /// The physical memory-mapped base address of the bootloader SMM firmware volume (FV).

```

Comment:
bootloader MM firmware volume

Reply:
if it is ok that line 39 name is different with description

Reply:
keep SMM as defined in FSPI ARCH UPD